shared/menubar/index.js

```diff
@@ -16,22 +14,20 @@ import {remote} from 'electron'
 import {ipcRenderer} from 'electron'
 import {loginTab} from '../constants/tabs'
 
-import type {Folder} from '../constants/types/flow-types'
-import type {FolderInfo} from './index.render'
+import type {Props as RenderProps} from './index.render'
 
-export type MenubarProps = {
+export type Props = {
   username: ?string,
-  folders: ?Array<Folder>,
+  folders: ?RenderProps,
   favoriteList: () => void,
-  openInKBFS: () => void,
+  openInKBFS: (target?: any) => void,
   loggedIn: ?boolean,
   switchTab: (tab: string) => void
-}
+} & RenderProps
```

Comment:
This type is a bit weird, why is RenderProps passed in to folders and also intersected (`& `)?